src/getDefaultDatabaseInstance.ts

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+import { getFirebaseProject } from "./management/projects";
+
+/**
+ * Tries to determine the default database instance for a project.
+ * @param options The command-line options object
+ * @returns The instance ID, empty if it doesn't exist.
+ */
+export async function getDefaultDatabaseInstance(options: any): Promise<string> {
+  const projectDetails = await getFirebaseProject(options.project);
+  if (projectDetails.resources && projectDetails.resources.realtimeDatabaseInstance) {
```

Comment:
You may be able to use `?` to simplify this a smidge (though it only works if it's pretty well typed):

```suggestion
  if (projectDetails.resources?.realtimeDatabaseInstance) {
```

even more, you may be able to do this? not sure if it's the safest though:

```typescript
return projectDetails.resources?.realtimeDatabaseInstance || "";
```